.github/ISSUE_TEMPLATE/improvement.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Improvement
+about: Suggest an improvement to the MetalK8s project
+labels: moonshot
+
+---
+<!-- Please only use this template for submitting improvement requests, and make sure it is linked to the relevant GitHub Project. -->
+
+**Component** (salt, containers, kubernetes, solutions, buildchain, tests…):
+
+**Why this is needed**:
+
+**What should be done**:
+
+**Implementation proposal** (strongly recommended):
+
+**Test plan/Acceptance criteria**:
```

Comment:
Both deserve their own section IMHO, but probably `Acceptance criteria` can be merged with `What should be done` (i.e. scope definition for the improvement).